src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -3919,5 +3919,1172 @@ class Test1 : I1
 
             ValidateIndexerImplementation_501(compilation1.SourceModule, "Test1");
         }
+
+        [Fact]
+        public void EventImplementation_101()
+        {
+            var source1 =
+@"
+public interface I1
+{
+    event System.Action E1 
+    {
+        add
+        {
+            System.Console.WriteLine(""add E1"");
+        }
+    }
+}
+
+class Test1 : I1
+{}
+";
+            var compilation1 = CreateCompilationWithMscorlib(source1, options: TestOptions.DebugDll,
+                                                             parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.Latest));
+            Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+            compilation1.VerifyDiagnostics(
+                // (4,25): error CS0065: 'I1.E1': event property must have both add and remove accessors
+                //     event System.Action E1 
+                Diagnostic(ErrorCode.ERR_EventNeedsBothAccessors, "E1").WithArguments("I1.E1").WithLocation(4, 25)
+                );
+
+            ValidateEventImplementationTest1_101(compilation1.SourceModule, haveAdd: true, haveRemove: false);
+
+            var source2 =
+@"
+class Test2 : I1
+{}
+";
+
+            var compilation2 = CreateCompilationWithMscorlib(source2, new[] { compilation1.ToMetadataReference() }, options: TestOptions.DebugDll);
+            Assert.True(compilation2.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
+
+            void Validate2(ModuleSymbol m)
+            {
+                ValidateEventImplementationTest2_101(m, haveAdd: true, haveRemove: false);
+            }
+
+            Validate2(compilation2.SourceModule);
+            compilation2.VerifyDiagnostics();
+            CompileAndVerify(compilation2, verify: false, symbolValidator: Validate2);
```

Comment:
`PEVerify` doesn't even handle C# 7 constructs. We should track that work, but I don't expect it will be in scope of completing and shipping this language feature.